### PR TITLE
Checkstyle license check - depend on  file from released branch

### LINKED ISSUE #166: Checkstyle license check depends on file in Github master branch
For Checkstyle plugin we reference file from our master branch. 
  
```
 <headerLocation>https://raw.github.com/codehaus-plexus/plexus-pom/master/src/main/resources/config/plexus-header.txt</headerLocation>
```  
If we remove this file - we may break existing builds

### PR DESCRIPTION
**Pull request by slachiewicz**
File in master can change or can be deleted

Closes #166

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -355,7 +355,8 @@ limitations under the License.
           <version>3.2.2</version>
           <configuration>
             <configLocation>config/maven_checks.xml</configLocation>
-            <headerLocation>https://raw.github.com/codehaus-plexus/plexus-pom/master/src/main/resources/config/plexus-header.txt</headerLocation>
+            <!-- version from release 10 referenced by branch -->
+            <headerLocation>https://github.com/codehaus-plexus/plexus-pom/blob/plexus-10/src/main/resources/config/plexus-header.txt</headerLocation>
```
Comment (slawekjaranowski) [inline pom.xml:359]: maybe spotless will be better here

Comment (slachiewicz) [reply pom.xml:359]: Agree that we should move to Spotless soon, but if someone will just remove plexus-header.txt file from master branch something very bad may happen ;-)

**Review by slawekjaranowski — APPROVED** [review]

**Comment by olamy** [comment]
If I understand correctly every project upgrading to this parent will fail if any single source file doesn't have this header?
Then how to fix the build?

**Comment by hboutemy** [comment]
oh, you're right `<headerLocation>https://raw.github.com/codehaus-plexus/plexus-pom/master/src/main/resources/config/plexus-header.txt</headerLocation>` is not really a safe value

notive that IIUC, we're switching to spotless for checks about this, isn't it?

**Pull request merged**